### DISCUSSION TITLE
Bigger case?

### DISCUSSION THREAD

**Discussion by olskar** [adonno/tagreader#111] [Q&A] [answered]
Hello! 

Im trying to build a tagreader, but all the buzzers I have are too big. Some kind of mini model of buzzer used? :)

Is it possible to get a STL with slightly more space @adonno  ? :)

**Comment by saboaua** [comment, marked as answer]
> Hello! 
> 
> 
> 
> Im trying to build a tagreader, but all the buzzers I have are too big. Some kind of mini model of buzzer used? :)
> 
> 
> 
> Is it possible to get a STL with slightly more space @adonno  ? :)

I created a bigger case to support dupon cables. 

![image](https://user-images.githubusercontent.com/33001915/138609151-1e7b92ed-018d-4d5a-8c45-637c2162aabd.jpeg)

Let me know if you need the stl file.

**Reply by olskar** [reply]
Id love the stl :) can it be uploaded here perhaps so other can use it aswell?

**Reply by saboaua** [reply]
Here is the stl file.

https://www.thingiverse.com/thing:5031307

**Comment by makuhlmann12** [comment]
Hi Olskar,

I had the same problem with my buzzer and just cut off some plastic at the upper end of the buzzer. Still works fine and fits into the case.
